exonum/src/events/codec.rs

```diff
@@ -40,12 +39,14 @@ impl MessagesCodec {
 }
 
 impl Decoder for MessagesCodec {
-    type Item = RawMessage;
-    type Error = io::Error;
+    type Item = UncheckedBuffer;
+    type Error = Error;
 
-    fn decode(&mut self, buf: &mut BytesMut) -> Result<Option<Self::Item>, io::Error> {
-        // Read header
-        if buf.len() < HEADER_LENGTH {
+    fn decode(&mut self, buf: &mut BytesMut) -> Result<Option<Self::Item>, Error> {
+        const U32_LENGTH: usize = 4;
+
+        // Read size
+        if buf.len() < U32_LENGTH {
```

Comment:
I think using `std::mem::size_of<u32>` is better, especially since it is used once in this function.